index.html

```diff
@@ -0,0 +1,524 @@
+      <div class="news">
+          <img src="images/default.png" alt="default image">
+          <div class="news-content">
+            <h3><a href="#">YouPorn now lets you create an account without an email</a></h3>
+            <span>Mix - 6 hours ago</span>
+          </div>
+      </div>
+      <div class="news">
+          <img src="images/default.png" alt="default image">
+          <div class="news-content">
+            <h3><a href="#">What does a copywriter do & how do you become one?</a></h3>
+            <span>TNW Deals - 7 hours ago</span>
+          </div>
+      </div>
+    </div>
+    <div class="surface">
+      <header>
+        <h1><a href="#">Microsoft Surface event 2019 <i class="fas fa-chevron-right"></i></a></h1>
+      </header>
+      <div class="surface-articles">
+        <img src="images/default.png" alt="default image">
+        <div class="surface-content">
+          <h3><a href="#">Surface Duo, Neo, and everything else at Microsoft's packed hardware event</a></h3>
+          <span>Napier Lopez - 5 days ago</span>
+        </div>
+      </div>
+      <div class="surface-articles">
+        <img src="images/default.png" alt="default image">
+        <div class="surface-content">
+          <h3><a href="#">It's real: Microsoft announces the Surface Duo, a dual-screen Android phone</a></h3>
+          <span>Ivan Mehta - 6 days ago</span>
+        </div>
+      </div>
+      <div class="surface-articles">
+        <img src="images/default.png" alt="default image">
+        <div class="surface-content">
+          <h3><a href="#">Microsoft teases Windows 10X, optimized for dual screens</a></h3>
+          <span>Rachel Kaser - 6 days ago</span>
+        </div>
+      </div>
+      <div class="surface-articles">
+        <img src="images/default.png" alt="default image">
+        <div class="surface-content">
+          <h3><a href="#">Microsoft's ARM-powered Surface Pro X takes on the iPad Pro</a></h3>
+          <span>Ivan Mehta - 6 days ago</span>
+        </div>
+      </div>
+    </div>
+    <div class="funding-rounds">
+      <header>
+        <a href="#">
+          <h1>Latest funding rounds on Index.co <i class="fas fa-chevron-right"></i></h1>
+        </a>
+        <ul>
+          <li><a href="#">Add info</a></li>
+          <li><a href="#">Get updates by email</a></li>
+          <li><a href="#">See more</a></li>
+        </ul>
+      </header>
+      <div class="funding-box">
+        <header>
+          <a href="#">
+            <span>Rickcast Solutions</span>
+            <div>
+              <p>$600k</p>
+              <p>SEED FUND</p>
+            </div>
+          </a>
+        </header>
+        <div class="funding-box-main">
+          <p><i class="fas fa-tag"></i> Project Management</p>
+          <p><i class="fas fa-briefcase"></i> Newark Venture Partners</p>
+        </div>
+        <footer>
+          <span>Riskcast Solutions Raises $600K in Pre-Seed Funding on FinSMEs</span>
+        </footer>
```

Comment:
The `<footer>` tag should be used for the footer of the page, please delete all the footer tags you have in the main content